### ISSUE TITLE
root index of all langs + snapshots

### ISSUE THREAD

**Issue by jbenet** [ipfs/distributed-wikipedia#3]
we should have a root "wikipedia on ipfs" index directory with:
- all languages
- all snapshots on ipfs (by date)
- any metadata we want
- (like dist.ipfs.io archive kinda)

**Comment by flyingzumwalt** [comment]
starting to gather the info for this root page at https://github.com/ipfs/distributed-wikipedia-mirror/blob/master/snapshot-hashes.md

**Comment by alzinging** [comment]
This YAML file shows last updates in 2021, we really need them updated to current snapshots and maintain a log of all the changed Qm Merkledags--as you suggested 6 years ago.

The YML was here when I got here a few months ago, but has been recently updated to be more easily visible. 

I also think we should run Ads on the whole thing to offset costs of storage and development of a Wiki on IPFS/Web3 framework that will interoperate with live Mediawiki/Wikipedia editing process, as they suggest in their "live mirror discussions." We could create a sort of conduit that will force upgrades of the Mediawiki software process to enable tiered administration of the editing and approval process. 

>> QUARRY on mwcloud.org looks interesting, though it seems to be "alpha" 

Can someone upload the "xml.tar.bz2" files from https://en.m.wikipedia.org/wiki/Wikipedia:Database_download

> TL;DR: GET THE MULTISTREAM VERSION! (and the corresponding index file, pages-articles-multistream-index.txt.bz2) in all languages available.  Wikisource.org also appears new. 

I am interested in developing a framework for Browser based rich text editing of Wiki pages, I think it might "spinoff" into an active community of Web3 open source blogger.com/medium.com writers and readers. The GO IPFS implementation Kubo appears to be in active development making "good strides" towards where we need to be to feel "multi cloud stable" across S3/Azure/Google with HA-sharding and FIL/Dolt 

https://docs.dolthub.com/introduction/what-is-dolt

They also have some "fresh good ideas" implemented and an active community of users. They are using a libp2p implementation of "swarm" that also appears to be in active hero status. I can't recall the name, something-gun. 

I am willing to work "Gratis" as in I will even support others development with modest funding. Please contact me if you are capable and willing. 

9546678083
getme.s.lamc.la

**Comment by kelson42** [comment]
@jbenet This is not what iofs.kiwix.org does?

**Comment by alzinging** [comment]
That site is using and references this project. Kiwix complicates the XML backup to "readable HTML" 

It's dated 9/1/2021. 

... though I wasn't going to backup the Wikipedia "image media" that adds another step. I am not familiar with Githubs automation, but I'm sure it works and will make this worthwhile. 

I have the Wikimedia "import to MySQL/HTML" process ... "understood" ... it won't take me long. 

I need help making a JavaScript client side wiki ... interface with IPFS. They are working on something like "git" level everything but editing in Kubo, I will probably use the IPFS JavaScript implementation instead, though, I imagine there is a client side git implementation also. I'll look.
